wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/privacy.html

```diff
@@ -4,21 +4,35 @@
 {% block content %}
     {% if page.id %}{% test_page_is_public page as is_public %}{% else %}{% test_page_is_public parent_page as is_public %}{% endif %}
 
-    {# The swap between public and private text is done using JS inside of privacy-switch.js when the response from the modal comes back #}
-    <div class="{% if not is_public %}w-hidden{% endif %}" data-privacy-sidebar-public>
-        {% trans 'Page visibility: ' as screen_reader_title_prefix %}
-        {% trans 'Visible to all' as title %}
-        {% trans 'Once live anyone can view' as help_text %}
-        {% with icon_name='view' %}
-            {{ block.super }}
-        {% endwith %}
-    </div>
-    <div class="{% if is_public %}w-hidden{% endif %}" data-privacy-sidebar-private>
-        {% trans 'Private' as title %}
-        {% trans 'Not visible to the public' as help_text %}
-        {% with icon_name='no-view' %}
-            {{ block.super }}
-        {% endwith %}
+    <div>
```

Comment:
how come you nested this within another div?

Reply:
Because i noticed that when changing privacy, shifts private div bit down:

https://github.com/user-attachments/assets/3b22e743-e065-4ead-9b71-4c3a449ff151

> Not wraping withing the div

https://github.com/user-attachments/assets/f5e1e031-ee64-45db-b7db-d7a4880710b8

> Wrapping withing div

https://github.com/user-attachments/assets/85d42780-ec30-4f88-9ab8-368930ba9d60

Reply:
Hmm ok. If that behaviour wasn't there before, maybe something else is the issue.

Can you do a bit more digging, it would be good to avoid having to nest this further.

It could be the ordering of the removal and adding of classes maybe.

Reply:
I noticed this when I first looked into the issue, and to confirm, I spin up the bakerydemo project on docker and the same thing is happening:

https://github.com/user-attachments/assets/efa3e748-f4b0-498e-ba43-fb3c518c544e

Reply:
It is causing by space-y-3 that being applied to display:none element (most probably) !!

So i created this custom css class in `client\scss\generic\_normalize.scss`:
```
.force-space-y-0 {
  margin-top: 0 !important;
  margin-bottom: 0 !important;
}
```
(lemme know if i should change the name or where to place)

and directly adding to those div:

` class="force-space-y-0 {% if not is_public %}w-hidden{% endif %}"`

`class="force-space-y-0 {% if is_public %}w-hidden{% endif %}"`

It is working as expacted !